moveit_ros/planning/planning_request_adapter_plugins/src/add_time_optimal_parameterization.cpp

```diff
@@ -46,11 +45,16 @@ using namespace trajectory_processing;
 /** @brief This adapter uses the time-optimal trajectory generation method */
 class AddTimeOptimalParameterization : public planning_request_adapter::PlanningRequestAdapter
 {
+  rclcpp::Logger LOGGER_ADD_OPTIMAL_PARAMETERIZATION =
```

Comment:
```suggestion
  static const rclcpp::Logger LOGGER =
```

Reply:
Changed all logging to their best practices.